src/pages/home/home.ts

```diff
@@ -239,7 +241,7 @@ export class HomePage {
               selectedPosition: selectedLocation,
             }
           );
-          this.locations = [];
+          // this.locations = [];
```

Comment:
is there a reason we were clearing locations here?

Reply:
After selecting a location, we clear the list so there is no more locations to select from. However, Joseph wants the user to be able to select another location if they don't like the one they selected. So removing this I guess for now is ok, since we're gonna have a story to select a another location.